examples/advanced_demo.py

```diff
@@ -176,29 +193,30 @@ def on_slider_changed(obj=None, event=-1):
         self.slider.set_value(self.chart.factor, 0)
         self.slider.set_event_cb(on_slider_changed)
 
-        
+
 class Screen_Main(lv.obj):
     def __init__(self, app, *args, **kwds):
         self.app = app
         super().__init__(*args, **kwds)
-        
+
         self.tabview = lv.tabview(self)
-        self.page_simple = Page_Simple(self.app, self.tabview.add_tab('Simple'))
-        self.page_buttons = Page_Buttons(self.app, self.tabview.add_tab('Buttons'))
-        self.page_chart = Page_Chart(self.app, self.tabview.add_tab('Chart'))
+        self.page_simple = Page_Simple(self.app, self.tabview.add_tab("Simple"))
+        self.page_buttons = Page_Buttons(self.app, self.tabview.add_tab("Buttons"))
+        self.page_chart = Page_Chart(self.app, self.tabview.add_tab("Chart"))
 
 
-class AdvancedDemoApplication():
+class AdvancedDemoApplication:
     def init_gui_SDL(self):
 
         import SDL
+
         SDL.init()
 
         # Register SDL display driver.
 
         disp_buf1 = lv.disp_buf_t()
-        buf1_1 = bytes(480*10)
-        lv.disp_buf_init(disp_buf1,buf1_1, None, len(buf1_1)//4)
+        buf1_1 = bytes(480 * 10)
+        lv.disp_buf_init(disp_buf1, buf1_1, None, len(buf1_1) // 4)
```

Comment:
This should also be changed to `lv.color_t.SIZE`, right?

Reply:
Correct.